package.json

```diff
@@ -58,17 +59,17 @@
     "clean:node": "rimraf index.js lib",
     "clean:es": "rimraf es",
     "clean:browser": "rimraf validator*.js",
-    "clean": "npm run clean:node && npm run clean:browser && npm run clean:es",
+    "clean": "run-p clean:*",
```

Comment:
Isn't the same result achieved using a single `&` and without having to add another dep?
```suggestion
    "clean": "npm run clean:node & npm run clean:browser & npm run clean:es",
```

Reply:
That doesn't work on Windows

Reply:
Damn, shitty cmd🙃